pkg/kdump/kdump-client.js

```diff
@@ -197,11 +197,11 @@ export class KdumpClient {
             target.target = "local";
 
         // "path" applies to all targets
-        // default to "/var/crash for "
+        // default to "/const/crash for "
         if ("path" in settings)
             target.path = settings.path.value;
         else if (["local", "ssh", "nfs", "mount"].indexOf(target.target) !== -1)
-            target.path = "/var/crash";
+            target.path = "/const/crash";
```

Comment:
hehe this is weird :)

Reply:
Haven't you heard? That's our exciting new read-only OS image! *tongue in cheek*

Thanks for spotting! Fixed.